platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -1376,9 +1406,10 @@ public AddressBook getAddressBook() {
     @SuppressWarnings("unchecked")
     @Override
     public <T extends SwirldState> AutoCloseableWrapper<T> getLatestImmutableState(@NonNull final String reason) {
```

Comment:
This method could now be annotated `@NonNull`

Reply:
done